libraries/AP_BoardConfig/AP_BoardConfig.h

```diff
@@ -91,7 +128,12 @@ class AP_BoardConfig
     } px4;
 
     static enum px4_board_type px4_configured_board;
-    
+
+#if HAL_WITH_UAVCAN
+    static int8_t _st_can_enable;
+    static int8_t _st_can_debug;
```

Comment:
can_debug is called from static everywhere and I think AP_Int8 can not be declared static. So I am using a transition variable filled in the start.